microprofile/reactive-messaging-smallrye/common/src/main/java/org/wildfly/microprofile/reactive/messaging/common/security/BaseReactiveMessagingSslConfigProcessor.java

```diff
@@ -0,0 +1,116 @@
+package org.wildfly.microprofile.reactive.messaging.common.security;
+
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+import org.jboss.as.server.deployment.AttachmentKey;
+import org.jboss.as.server.deployment.Attachments;
+import org.jboss.as.server.deployment.DeploymentPhaseContext;
+import org.jboss.as.server.deployment.DeploymentUnit;
+import org.jboss.as.server.deployment.DeploymentUnitProcessingException;
+import org.jboss.as.server.deployment.DeploymentUnitProcessor;
+import org.jboss.modules.Module;
+import org.jboss.msc.service.ServiceName;
+import org.wildfly.microprofile.reactive.messaging.config.ReactiveMessagingConfigSource;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import static org.wildfly.microprofile.reactive.messaging.common.ReactiveMessagingAttachments.IS_REACTIVE_MESSAGING_DEPLOYMENT;
+import static org.wildfly.microprofile.reactive.messaging.common._private.MicroProfileReactiveMessagingCommonLogger.LOGGER;
+
+public abstract class BaseReactiveMessagingSslConfigProcessor implements DeploymentUnitProcessor {
+
+    private static final String RM_KAFKA_GLOBAL_PROPERTY_PREFIX = "mp.messaging.connector.";
```

Comment:
Not sure whether `KAFKA` word in this constant isn't confusing with also `AMQP` supported now? What about just `RM_CONNECTOR_PROPERTY_PREFIX` or `RM_GLOBAL_PROPERTY_PREFIX`?